src/_png.cpp

```diff
@@ -139,6 +135,9 @@ const char *Py_write_png__doc__ =
     "    Byte string containing the PNG content if None was passed in for\n"
     "    file, otherwise None is returned.\n";
 
+// this code is heavily adapted from the paint license, which is in
```

Comment:
While we're here, let's fix the sentence: "This code is heavily adapted from code licensed via the BSD-compatible LICENSE_PAINT file included in this distribution."   Or something like that.

Reply:
@efiring pushed a re-wording.